study-builder/tenants/testboston/pages/login.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>Sign In with Auth0</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+</head>
+  <style>
+    body, html {
+      height: 100%;
+      background-color: #f9f9f9;
+    }
+
+    .login-container {
+      position: relative;
+      height: 100%;
+    }
+
+    .login-box {
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%);
+      padding: 15px;
+      background-color: #fff;
+      box-shadow: 0px 5px 5px #ccc;
+      border-radius: 5px;
+      border-top: 1px solid #e9e9e9;
+    }
+
+    .login-header {
+      text-align: center;
+    }
+
+    .login-header img {
+      width: 75px;
+    }
+
+    #error-message {
+      display: none;
+    }
+  </style>
+<body>
+      <script src="https://angio.dev.datadonationplatform.org/assets/config/pepperConfig.js"></script>
```

Comment:
Oops, I see my  "Can we load stuff into universal login page?" experiment has left some junk...

Reply:
Oh, then in that case I'll just copy over the one from CMI so we have something known-good to start with.